docs/eventing/getting-started.md

```diff
@@ -19,299 +21,377 @@ Use this guide to learn how to create, send, and verify events in Knative. The s
+- A
+  [Kubernetes cluster](https://kubernetes.io/docs/concepts/cluster-administration/cluster-administration-overview/)
+  running v1.15 or higher.
 
-- [`kubectl` CLI tool](https://kubernetes.io/docs/reference/kubectl/overview/) within a minor version of your Kubernetes cluster.
+- [`kubectl` CLI tool](https://kubernetes.io/docs/reference/kubectl/overview/)
+  within a minor version of your Kubernetes cluster.
 
 - [curl v7.65 or higher](https://curl.haxx.se/download.html).
 
 - Knative Eventing component.
 
   - Knative Eventing in-memory channel.
 
-**Important Note:** Some Knative Eventing features do not work when using Minikube due to [this](https://github.com/kubernetes/minikube/issues/1568) bug. For local testing you can use [kind](https://github.com/kubernetes-sigs/kind).
-
+**Important Note:** Some Knative Eventing features do not work when using
+Minikube due to [this](https://github.com/kubernetes/minikube/issues/1568) bug.
+For local testing you can use [kind](https://github.com/kubernetes-sigs/kind).
 
 ### Installing Knative Eventing
 
-If you previously [created a Knative cluster](../install), you might already have Knative Eventing installed and running. You can check to see if the Eventing component exists on your cluster by running:
+If you previously [created a Knative cluster](../install), you might already
+have Knative Eventing installed and running. You can check to see if the
+Eventing component exists on your cluster by running:
 
 ```sh
 kubectl get pods --namespace knative-eventing
 ```
 
-If the `knative-eventing` namespace or the `imc-controller-*` does not exist, use the following steps to install Knative Eventing with the in-memory channel:
+If the `knative-eventing` namespace or the `imc-controller-*` does not exist,
+use the following steps to install Knative Eventing with the in-memory channel:
 
-1. Make sure that you have a functioning Kubernetes cluster. See the [Comprehensive Install guide](../install) for more information.
+1. Make sure that you have a functioning Kubernetes cluster. See the
+   [Comprehensive Install guide](../install) for more information.
 
-   - Old versions of Knative Serving doesn't necessarily work well with latest Knative Eventing, so try to install the latest version of Knative Serving.
+   - Old versions of Knative Serving doesn't necessarily work well with latest
+     Knative Eventing, so try to install the latest version of Knative Serving.
 
-   - If your Kubernetes cluster comes with pre-installed Istio, make sure it has `cluster-local-gateway` [deployed](https://github.com/knative/serving/blob/master/DEVELOPMENT.md#deploy-istio). Depending on which Istio version you have, you'd need to apply the `istio-knative-extras.yaml` in the corresponding version folder at [here](https://github.com/knative/serving/tree/{{< branch >}}/third_party).
+   - If your Kubernetes cluster comes with pre-installed Istio, make sure it has
```

Comment:
would you mind adding this ti 'any-kubernetes-cluster.md' section that talks about eventing install.